# Description: Use ruby1.8 specifically as Chef doesn't support ruby1.9 yet.
Index: chef-0.8.6/chef-solr/bin/chef-solr
===================================================================
--- chef-0.8.6.orig/chef-solr/bin/chef-solr	2010-03-07 01:39:44.000000000 -0700
+++ chef-0.8.6/chef-solr/bin/chef-solr	2010-03-07 01:39:50.000000000 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby1.8
 #
 # Author:: Adam Jacob (<adam@opscode.com>)
 # Copyright:: Copyright (c) 2009 Opscode, Inc.
Index: chef-0.8.6/chef-solr/bin/chef-solr-indexer
===================================================================
--- chef-0.8.6.orig/chef-solr/bin/chef-solr-indexer	2010-03-07 01:39:55.000000000 -0700
+++ chef-0.8.6/chef-solr/bin/chef-solr-indexer	2010-03-07 01:40:04.000000000 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby1.8
 #
 # Author:: Adam Jacob (<adam@opscode.com>)
 # Copyright:: Copyright (c) 2009 Opscode, Inc.
Index: chef-0.8.6/chef-solr/bin/chef-solr-rebuild
===================================================================
--- chef-0.8.6.orig/chef-solr/bin/chef-solr-rebuild	2010-03-07 01:40:10.000000000 -0700
+++ chef-0.8.6/chef-solr/bin/chef-solr-rebuild	2010-03-07 01:40:18.000000000 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby1.8
 #
 # Author:: Adam Jacob (<adam@opscode.com>)
 # Copyright:: Copyright (c) 2009 Opscode, Inc.
